src/clients/TorrentClient.ts

```diff
@@ -18,14 +19,16 @@ export interface TorrentClient {
 }
 
 function instantiateDownloadClient() {
-	const { rtorrentRpcUrl, qbittorrentUrl, transmissionRpcUrl } =
+	const { rtorrentRpcUrl, qbittorrentUrl, transmissionRpcUrl, delugeWebUrl, delugeWebPassword } =
```

Comment:
is it a limitation of deluge that requires the weburl and password to be treated separately unlike the other clients?

Reply:
For deluge's WebUI, the page itself does not require any standard authentication header. Instead, we are confronted with a form for the password which is pushed for authorization with the `auth.login` RPC method.

This auth is done for the /json endpoint as well, but "manually" since its a JSON-RPC endpoint.

I'm determining success by whether `result` in the returned JSON for the `data` block is true.

Reply:
FWIW, neither do any of the other torrent clients (maybe transmission does? idr). I set it up that way so as to have fewer options. In hindsight it hasn't been great UX (having to url encode your password is obviously stupid) but here we are

Reply:
Addressed mostly in (https://github.com/cross-seed/cross-seed/pull/499/commits/39145621296eb067e346d1f4b9c97d82d3200193) I believe.